src/Compilers/CSharp/Test/Symbol/Symbols/Source/PropertyTests.cs

```diff
@@ -2849,21 +2849,17 @@ public void Bug1073332_02()
             var text = @"
 unsafe class Test
 {
-    int[] property { get; } = stackalloc int[256];
```

Comment:
it was just to make the test less noisy.  Because there isn't a parse error now, we end up doing more binding.  and we were reporting an error converting an int* to an int[].  So this was just to clean that up a bit.